packages/core/stories/menu/menu.qa.stories.css

```diff
@@ -0,0 +1,7 @@
+.singleLevel .background-block {
+  height: 500px;
+}
+
+.saltQAContainer.singleLevel {
+  --qaContainer-item-width: 160px !important;
```

Comment:
At the next opportunity, worth looking at QA container to avoid having to publicly use !important.